### DISCUSSION TITLE
I cant'e see exported files on my C drive although I see them in LC

### DISCUSSION THREAD

**Discussion by rochocki7** [mifi/lossless-cut#1921] [Q&A]
Hi There.
I was looking for a program to losslessly trim mp4 and found Losless Cut. All videos on YT show how after exporting you have a new file with a trimmed video in the same output folder. But I don't have IT??? When opening in LC there it is but It is not in the output Folder when opening it directly. Any ideas what I need to do?

**Comment by mifi** [comment]
not sure what you mean by that exactly. got some screenshots? you can select working directory in losslesscut (top right), which is where exported files will be written to.
